terraform_env/production/outputs.tf

```diff
@@ -0,0 +1,12 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
+}
```

Comment:
The webhook URL is also sensitive - it contains a deployment password (`azurerm_app_service.main.site_credential[0].password`).
So you should mark this output with sensitive = true. Otherwise it gets logged in Travis after terraform apply completes:
https://app.travis-ci.com/github/kiranuppi/DevOps-Course-Starter/jobs/554847009